server/src/test/java/org/opensearch/cluster/coordination/ClusterBootstrapServiceTests.java

```diff
@@ -663,23 +653,6 @@ public void testBootstrapsAutomaticallyWithSingleNodeDiscovery() {
         assertFalse(bootstrapped.get()); // should only bootstrap once
     }
 
-    public void testFailBootstrapWithBothSingleNodeDiscoveryAndInitialMasterNodes() {
-        final Settings.Builder settings = Settings.builder()
-            .put(DiscoveryModule.DISCOVERY_TYPE_SETTING.getKey(), DiscoveryModule.SINGLE_NODE_DISCOVERY_TYPE)
-            .put(NODE_NAME_SETTING.getKey(), localNode.getName())
-            .put(INITIAL_MASTER_NODES_SETTING.getKey(), "test");
-
-        assertThat(
-            expectThrows(
-                IllegalArgumentException.class,
-                () -> new ClusterBootstrapService(settings.build(), transportService, () -> emptyList(), () -> false, vc -> fail())
-            ).getMessage(),
-            containsString(
-                "setting [" + INITIAL_MASTER_NODES_SETTING.getKey() + "] is not allowed when [discovery.type] is set " + "to [single-node]"
-            )
-        );
-    }
-
```

Comment:
Note: These 2 tests in this class (`ClusterBootstrapServiceTests`) is moved to the new class `ClusterBootstrapServiceDeprecatedMasterTests`